adding my blog (plz test first i cant rn) (#20)
RMIT-BattleBots/rmit-battlebots.github.io: content/blog/ARC, what are we?.md

Title: ARC, what are we?  
Date: 2026-01-30  
Author: Romi Luna Cardoso  

> Excerpt: ARC's goal is  to design and create combat robots with the goal to compete with them in a variety of competitions and represent the school.  

Reviewer: enclose the content text in quotation marks ""
```suggestion
Title: "ARC, what are we?"
Date: "2026-01-30"
Author: "Romi Luna Cardoso"
Excerpt: "ARC's goal is  to design and create combat robots with the goal to compete with them in a variety of competitions and represent the school."
```

> Excerpt: ARC's goal is  to design and create combat robots with the goal to compete with them in a variety of competitions and represent the school.  

Reviewer: title, date, author and exerpt must be in lower case

```suggestion
title: ARC, what are we?  
date: 2026-01-30  
author: Romi Luna Cardoso  
excerpt: ARC's goal is  to design and create combat robots with the goal to compete with them in a variety of competitions and represent the school.  
```

> 

Reviewer: youll need to add
`featuredImage: "/blog-images/XXXXXXX.jpg"`

# ARC, what are we?  

ARC's (Advanced RMIT Combat-Robotics) goal is to design and create combat robots with the goal to compete at a variety of competitions and  
represent the school.  

## What is ARC to me?  

ARC to me is all about the engineering processes and learning from it. It is immensely satisfying to watch a project from start as just an idea on   
a whiteboard and to see it evolve through many iterations ending up as a thing you can hold in your hands and watch compete. And even then as you   
are watching it do the thing you designed it to do, your still thinking about all the improvements you could make and the way you can do them. It   

> is truely one of the best things in this world.  

Reviewer: ```suggestion
is truly one of the best things in this world.  
```

![action shot of dave](/blog-images/robo-action-shot.png)

## What is it like to be part of ARC?  

ARC is a very welcoming environment for anyone from any background, experience level, or identity. We hope to a fun and safe learning environment so   
you can meet and make friends with like-minded individuals - and make cool robots along the way. ARC is a much more laid back club in comparison to   

> others here at RMIT so there is no pressure to carve out en enormous amount of time for the club. But of cource the more time you put into it the   

Reviewer: ```suggestion
others here at RMIT so there is no pressure to carve out an enormous amount of time for the club. But of course the more time you put into it the   
```

more you will get out of it.  

![The old clubspace with people working in it](/blog-images/club-at-work-photo.jpg)

> ## What recorces are available at ARC?  

Reviewer: ```suggestion
## What resources are available at ARC?  
```

> We of cource have access to a number of CADs (Computor-aided Design tools) thanks to RMIT. We also have access to the *Maker Space*, though with the   

Reviewer: ```suggestion
We of cource have access to a number of CADs (Computer-aided Design tools) thanks to RMIT. We also have access to the *Maker Space*, though with the   
```

> acquisition of a new *semi* club space we are looking to aquire some 3d printers for our own. This will allow us to use own own filliments and have   

Reviewer: ```suggestion
acquisition of a new *semi* club space we are looking to acquire some 3d printers for our own. This will allow us to use our own filaments and have   
```

> quicer access for all printing jobs. Aditonaly we have acces to a small wood workshop in the old club space and we have some contacts in the Bundoora   

Reviewer: ```suggestion
quicker access for all printing jobs. Additionally we have access to a small wood workshop in the old club space and we have some contacts in the Bundoora   
```
